bin/GameIndex.dbf

```diff
@@ -4617,14 +4618,8 @@ Region = NTSC-U
 Serial = SCUS-97150
 Name   = Formula One 2001
 Region = NTSC-U
-Compat = 3
+Compat = 5
 XgKickHack = 1
-[patches = 272F1C14]
-	comment=patches by Plot and Ref Skip videos
-	patch=0,EE,0010aafc,word,00000000
-	patch=0,EE,0010ae2c,word,00000000
-	patch=0,EE,00230010,word,00000000
-[/patches]
```

Comment:
Was this patch even necessary after the 0.9.X versions? The only issue on PAL version video's was just that they were black screens with hardware renderers until the blackFMV PR.